src/sentry/demo/data_population.py

```diff
@@ -87,55 +105,264 @@ def generate_user():
+                trace_id = uuid4().hex
+
+                old_span_id = react_transaction["contexts"]["trace"]["span_id"]
+                frontend_root_span_id = uuid4().hex[:16]
+                frontend_duration = random_normal(2000 - 50 * day, 250, 1000) / 1000.0
+
+                frontend_context = {
+                    "trace": {
+                        "type": "trace",
+                        "trace_id": trace_id,
+                        "span_id": frontend_root_span_id,
+                    }
+                }
+
+                # React transaction
+                local_event = copy.deepcopy(react_transaction)
+                local_event.update(
+                    project=react_project,
+                    platform="javascript-transaction",
+                    event_id=uuid4().hex,
+                    user=transaction_user,
+                    timestamp=timestamp,
+                    # start_timestamp decreases based on day so that there's a trend
+                    start_timestamp=timestamp - timedelta(seconds=frontend_duration),
+                    measurements={
+                        "fp": {"value": random_normal(1250 - 50 * day, 200, 500)},
+                        "fcp": {"value": random_normal(1250 - 50 * day, 200, 500)},
+                        "lcp": {"value": random_normal(2800 - 50 * day, 400, 2000)},
+                        "fid": {"value": random_normal(5 - 0.125 * day, 2, 1)},
+                    },
+                    # Root
+                    parent_span_id=None,
+                    span_id=frontend_root_span_id,
+                    trace=trace_id,
+                    contexts=frontend_context,
+                )
+
+                fix_spans(local_event, old_span_id)
+                safe_send_event(local_event)
+
+                # React error
+                local_event = copy.deepcopy(react_error)
+                local_event.update(
+                    project=react_project,
+                    platform=react_project.platform,
+                    timestamp=timestamp,
+                    user=transaction_user,
+                    contexts=frontend_context,
+                )
+                fix_breadrumbs(local_event)
+                safe_send_event(local_event)
+
+                # python transaction
+                old_span_id = python_transaction["contexts"]["trace"]["span_id"]
+                backend_duration = random_normal(1500 + 50 * day, 250, 500)
+
+                backend_context = {
+                    "trace": {
+                        "type": "trace",
+                        "trace_id": trace_id,
+                        "span_id": uuid4().hex[:16],
+                    }
+                }
+
+                local_event = copy.deepcopy(python_transaction)
+                local_event.update(
+                    project=python_project,
+                    platform="transaction",
+                    timestamp=timestamp,
```

Comment:
nit: I still need to update this in `bin/load-mocks` but should subtract some amount from the backend transaction too, cause all the transactions end at the same time within the trace
<img width="894" alt="image" src="https://user-images.githubusercontent.com/4205004/111555045-ea724e00-875d-11eb-917a-17335435a91e.png">

Reply:
@wmak I don't think we need to be _that_ realistic, though we can revisit this in the future if we want.